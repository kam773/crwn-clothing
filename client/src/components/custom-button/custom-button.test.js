import React from 'react';
import { shallow } from 'enzyme';
import CustomButton from './custom-button.component';
import renderer from 'react-test-renderer';

describe('create snapshot', () => {
  it('captures snapshot of CustomButton', () => {
    const wrapper = renderer.create(<CustomButton />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});

describe('shallow render react component', () => {
  it('renders DUMB react component', () => {
    const wrapper = shallow(<CustomButton />);
    expect(wrapper.length).toEqual(1);
  });
});
