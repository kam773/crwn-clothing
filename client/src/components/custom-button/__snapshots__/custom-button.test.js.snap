// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot captures snapshot of CustomButton 1`] = `
<button
  className="sc-bdVaJa kqgmkj"
/>
`;
